packages/domain-picker/src/hooks/use-domain-suggestions.ts

```diff
@@ -22,15 +22,17 @@ export function useDomainSuggestions(
 	searchTerm = '',
 	quantity: number,
 	domainCategory?: string,
-	locale = 'en'
+	locale = 'en',
+	extraOptions = {}
 ): DomainSuggestionsResult | undefined {
 	const [ domainSearch ] = useDebounce( searchTerm, DOMAIN_SEARCH_DEBOUNCE_INTERVAL );
-
-	// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-	// @ts-ignore
 	// Missing types for invalidateResolutionForStoreSelector
 	// (see packages/data/src/namespace-store/metadata/actions.js#L57)
```

Comment:
We can probably remove this line too (I checked that file path, and it's not even correct anymore)